mandatory/1-fix-functions.js

```diff
@@ -21,18 +19,18 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num;
```

Comment:
while this work, I think it might be nicer if we do `let isBigEnough = num > 10`, normally if a variable name read like `isXXX` it indicates it is a boolean value